src/Telemetry/AzureDevOpsTimeline/KustoTimelineTelemetryRepository.cs

```diff
@@ -44,14 +44,19 @@ public KustoTimelineTelemetryRepository(ILogger<KustoTimelineTelemetryRepository
             _database = options.Value.Database;
         }
 
-        public async Task<DateTimeOffset?> GetLatestTimelineBuild(string project)
+        public async Task<DateTimeOffset?> GetLatestTimelineBuild(AzureDevOpsInstance instance)
         {
             try
             {
                 using IDataReader result = await _query.ExecuteQueryAsync(
                     _database,
                     // This isn't use controlled, so I'm not worried about the Kusto injection
-                    $"TimelineBuilds | where Project == '{project}' | summarize max(FinishTime)",
+                    $"""
+                    TimelineBuilds
+                    | where coalesce(Organization, "dnceng") == '{instance.Organization}'
```

Comment:
This _could_ turn into an injection attack.  Would it be possible to parameterize this? I know right now they are controlled by configuration... but it's easy for code like this to move/duplicate into places where it's not safe.

Reply:
I should be able to